src/middlewared/middlewared/plugins/activedirectory.py

```diff
@@ -297,6 +297,30 @@ async def common_validate(self, new, old, verrors):
         Bool('enable'),
         update=True
     ))
+    @returns(Dict(
```

Comment:
We should probably add `ENTRY` variable like other plugins do and use it instead of copy-paste